breakout_3_Observability-with-AI/breakout_3_answers/3.1-grafana-assistant.md

```diff
@@ -0,0 +1,9 @@
+## Grafana AI
+Q: What languages are used for my app deployed in that namespace?
+
+A: Go, Java
```

Comment:
@Alex3k - most of the languages are not getting picked up, any thoughts on how we can improve this query?

Reply:
Interesting, it found Java, python, .net, php, go, javascript for me 


Maybe we could tell it to look at process run time or telemetry sdk in the log data or trace data?